src/components/flex/Flex.tsx

```diff
@@ -42,7 +61,9 @@ export const Flex = ({
     style={{
       ...style,
       '--dashboard-flex-wrapper-align-items': alignItems,
+      '--dashboard-flex-wrapper-flex-children': childrenFlex,
       '--dashboard-flex-wrapper-flex-direction': flexDirection,
+      '--dashboard-flex-wrapper-flex-wrap': flexWrap,
       '--dashboard-flex-wrapper-gap': `${gap}px`,
       '--dashboard-flex-wrapper-justify-content': justifyContent,
       '--dashboard-flex-wrapper-padding': `${padding || 0}px`,
```

Comment:
What do you think about setting the default in the function parameters like the rest? (I know I was probably the one that did this in the first place)

Reply:
Didn't see that. Good catch